lib/dstu2_resources.rb

```diff
@@ -744,6 +756,17 @@ module Resources
               ],
               "text": "Pyogenic Arthritis Involving Multiple Sites"
             },
+            "category": {
+              "coding": [
+                {
+                  "system": "http://argonaut.hl7.org/extension-codes",
```

Comment:
According to Argonaut doc [1], this should be "http://hl7.org/fhir/argo-condition-category". Defect?

[1] http://argonautwiki.hl7.org/index.php?title=Problems